pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "pytroll-collectors"
+dynamic = ["version"]
+description = "Pytroll data collectors"
+readme = "README.md"
+license = {file = "LICENSE"}
```

Comment:
Double check but I don't think this is the newest way to do this. For 1, you need to remove the license classifier in `classifiers`. Let me check what I did for other packages...

Reply:
yes, I read the spec, and it's clear the sample project is outdated. Fixed in the latest commit.